src/Razor/test/Microsoft.VisualStudio.Razor.IntegrationTests/CodeFoldingTests.cs

```diff
@@ -37,6 +36,7 @@ private async Task AssertFoldableBlocksAsync(params string[] blockTexts)
 
             var foldableSpans = blockTexts.Select(blockText =>
             {
+                Assert.Contains(blockText, text);
```

Comment:
Added this because I definitely got the text wrong a few times in my arguments and this made it much easier to diagnose

Reply:
More Asserts more better.